docs/ext/commands/help.rst

```diff
@@ -0,0 +1,78 @@
+.. currentmodule:: discord
+
+.. _ext_commands_help:
+
+Help Commands
+=================
+
+The simplicity of :class:`~.commands.HelpCommand` allows you to customise
+your experience very easily.
```

Comment:
This sentence seems like it belongs somewhere explaining how to do custom commands. Which... this does not do. Probably re-order the entire structure. To something like this:

1. Using The Help Command
    - Probably talk about the interface of the actual help command
      show examples etc
    - Talk about the sending help with ctx here
2. Replacing The Help Command
    - Talk about how you can set `bot.help_command`
        - This would be a good time to show `commands.MinimalHelpCommand`
    - Talk about how you can do `bot.help_command.cog = ...`
3. Writing A Help Command
    - Talk about the steps to write your own custom help
    - Describe all of the member functions required to make a help command and walk through the steps together

This is what I had in mind as a rough idea. Documentation of prose pages requires planning and working with flow so the contents are coherent instead of random paragraphs.